src/globals/Calc.ts

```diff
@@ -64,7 +73,21 @@ export type DispatchedEvent =
         fromTextMode?: boolean;
       };
       state: GraphState;
-    };
+    }
+  | {
+      // Note: this has more parameters. I just haven't found a need for them yet.
+      type: "set-item-latex";
+      latex: string;
+      id: string;
+    }
+  | {
+      type: "on-special-key-pressed";
+      key: string;
+      // used in compact-view plugin
```

Comment:
different plugin? Or is this unused?

Reply:
Yeah, that's for a different plugin (though the event type itself is part of base Desmos). Intellisense doesn't use it.